mindsdb_native/libs/controllers/predictor.py

```diff
@@ -246,6 +246,7 @@ def learn(self,
                 data_split_indexes = advanced_args.get('data_split_indexes', None),
                 tags_delimiter = advanced_args.get('tags_delimiter', ','),
                 force_predict = advanced_args.get('force_predict', False),
+                mixer_class = advanced_args.get('mixer_class', None),
```

Comment:
Set this to be NNMixer + Sklearn mixer for now, since the former should be quick to train and give us a lot of potential gains on many datasets.

Reply:
Also remove `None` handling from lightwood.py and just handle it here.

Reply:
> Set this to be NNMixer + Sklearn mixer for now, since the former should be quick to train and give us a lot of potential gains on many datasets.

providing `mixer_class` forces usage of a single mixer